File: Day 45/Classwork/task1.py
'''შევქმნათ ფუნქცია რომელიც გადმოცემული სტრინგის გაატმაგებულ ვერსიას დააბრუნებს'''


def myfunc(s):
    new_s=""
    for i in range(10):
        new_s+=s
    return new_s

print(myfunc("saba"))

File: Day 045/Classwork/task1.js
//შევქმნათ ფუნქცია რომელიც გადმოცემული სტრინგის გაატმაგებულ ვერსიას დააბრუნებს

function myfunc(s){
    let new_s=""
    for(let i=1; i<=10; i++){
        new_s+=s
    }
    return new_s
}

console.log(myfunc("saba"))